zerver/tests/test_bots.py

```diff
@@ -907,3 +907,32 @@ def test_patch_bogus_bot(self):
         result = self.client_patch("/json/bots/nonexistent-bot@zulip.com", bot_info)
         self.assert_json_error(result, 'No such user')
         self.assert_num_bots_equal(1)
+
+    def test_create_outgoing_webhook_bot(self, **extras):
+        # type: (**Any) -> None
+        self.login(self.example_email('hamlet'))
+        bot_info = {
+            'full_name': 'Outgoing Webhook test bot',
+            'short_name': 'outgoingservicebot',
+            'bot_type': 3,
```

Comment:
This should be UserProfile.OUTGOING_WEBHOOK_BOT.  No need to use the magic number 3.